src/oss/python/integrations/document_loaders/azure_blob_storage.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Azure Blob Storage Loader
+---
+
+>[Azure Blob Storage](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction) is Microsoft's object storage solution for the cloud. Blob Storage is optimized for storing massive amounts of unstructured data. Unstructured data is data that doesn't adhere to a particular data model or definition, such as text or binary data.
+
+`Azure Blob Storage` is designed for:
+
+- Serving images or documents directly to a browser.
+- Storing files for distributed access.
+- Streaming video and audio.
+- Writing to log files.
+- Storing data for backup and restore, disaster recovery, and archiving.
+- Storing data for analysis by an on-premises or Azure-hosted service.
+
+This notebook covers how to load document objects from a container on `Azure Blob Storage`. For more detailed documentation on the document loader, see the [Azure Blob Storage Loader API Reference](https://reference.langchain.com/python/integrations/langchain_azure_storage/).
+
+> [!NOTE]
+> This document loader replaces the previous [`AzureBlobStorageFileLoader`](https://python.langchain.com/api_reference/community/document_loaders/langchain_community.document_loaders.azure_blob_storage_file.AzureBlobStorageFileLoader.html) and [`AzureBlobStorageContainerLoader`](https://python.langchain.com/api_reference/community/document_loaders/langchain_community.document_loaders.azure_blob_storage_container.AzureBlobStorageContainerLoader.html) from `langchain_community`. It is recommended to use this new loader for better flexibility and customization when loading documents from Azure Blob Storage. For detailed instructions on migrating to the new loader, refer to the [migration guide](https://github.com/langchain-ai/langchain-azure/blob/main/libs/azure-storage/README.md#migrating-from-langchain-community-azure-storage-document-loaders)
+
+
+## Setup
+
+```python
+pip install -qU langchain-azure-storage
+```
+
+```python
+from langchain_azure_storage.document_loaders import AzureBlobStorageLoader
+```
+
+## Load from container
+
+```python
+loader = AzureBlobStorageLoader(
+    account_url="https://<storage-account-name>.blob.core.windows.net",
+    container="<container-name>",
+)
+
+for doc in loader.load():
+    print(doc)
+```
+
+```output
+[Document(page_content='Lorem ipsum dolor sit amet.', lookup_str='', metadata={'source': '/var/folders/y6/8_bzdg295ld6s1_97_12m4lr0000gn/T/tmpaa9xl6ch/fake.docx'}, lookup_index=0)]
+```
+
+You can also specify a prefix for more fine-grained control over what files to load.
+
+```python
+loader = AzureBlobStorageLoader(
+    account_url="https://<storage-account-name>.blob.core.windows.net",
+    container="<container-name>",
+    prefix="<prefix>",
+)
+```
+
+## Load from container by blob name
+
+You can load documents from a list of blob names, which uses only the blobs provided instead of a call to list blobs.
```

Comment:
Let's add "API" to right before call to make it clear we are avoiding an API hop in this example.